### PR TITLE
[AMS-1669] Different ImageSharp memory pool settings for different environments

### PR DESCRIPTION
**Pull request by denisivan0v**

### CODE REVIEW

File: src/VStore.Renderer/Startup.cs [hunk 1]
```diff
@@ -176,11 +176,21 @@ public void ConfigureContainer(ContainerBuilder builder)
                        (parameterInfo, context) => context.Resolve<ICephS3Client>())
                    .SingleInstance();
             builder.RegisterType<RawFileStorageInfoProvider>().SingleInstance();
-            builder.RegisterType<MemoryBasedRequestLimiter>().SingleInstance();
             builder.RegisterType<MetricsProvider>().SingleInstance();
+
+            if (!_environment.IsProduction())
```
Comment (xakep139) [inline src/VStore.Renderer/Startup.cs]: Remove negation and swap condition statements

Comment (denisivan0v) [reply src/VStore.Renderer/Startup.cs]: We have the same order in `Configure`, so I would like to have this check similar to the existing one.

**Comment by denisivan0v** [comment]
@xakep139 what do you think about memory consumption tests? Now we have different app behaviour for different environments, so now it much less determinism in memory allocations even than it was before. Should we have these tests disabled or it's better just to delete them?

**Comment by xakep139** [comment]
I suppose we could use them in future memory tweaks of renderer

**Pull request merged**